jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -0,0 +1,18 @@
+from glue.core import BaseData
+from glue_jupyter.bqplot.profile import BqplotProfileView
+from specutils import Spectrum1D
+
+from jdaviz.core.registries import viewer_registry
+
+__all__ = ['SpecvizProfileView']
+
+
+@viewer_registry("specviz-profile-viewer", label="Profile 1D (Specviz)")
+class SpecvizProfileView(BqplotProfileView):
```

Comment:
Why is this sub-class needed? I think if possible we want to avoid a specialized viewer here, since it then implies we need specialized viewers in cubeviz/mosviz (not the end of the world, but perhaps confusing)

Reply:
I modeled it off of how cubeviz worked, so I actually figured it was the paradigm we were working with. But I might have misinterpreted your message of "just like the bottom half of cubeviz" sentiment; please correct me if I'm wrong!

Reply:
Subclassing these viewers is how we define the default class that retrieved data gets converted to when using the api. E.g. in the `cubeviz-profile-viewer` viewer class, we have the `default_class` set to `Spectrum1D` such that when a user does `app.get_data_from_viewer('spectrum-viewer')`, the returned data object is a `Spectrum1D`. This isn't necessary for every viewer, only those in which we want to implicitly convert the data to a particular data object.

Reply:
Aha, thanks for clarifying that @nmearl.  In that case here's my suggestion:

1. For now, use a *single* custom viewer in both specviz and cubeviz (and we'll use the same in mosviz) - I suggest actually calling it the `SpecvizProfileView`, as it's easier to explain to users as "the spectrum view in cubeviz/mosviz is a specviz view", even if that's being a bit fast and loose with the details.
2. In the future we may want to move this to live in `glue-astronomy` since it's not really `jdaviz`-specific, but we can deal with that later. (As an *alternative* upstreaming approach we might have the config file have this information instead of the code.  But I could we can discuss that later.)

Sound reasonable, @nmearl and @dtn5ah?

Reply:
Yep that seems reasonable to me. I hesitate just in case we might one day need specific functionality for one vs the other later on, but we can always subclass then, if ever. I figure we should probably form a new ticket for this?

Reply:
Just want to post here for posterity: this makes specviz a dependency of both cubeviz and mosviz (which is fine, especially if want to support doing spectral-type things in either of the latter apps which duplicating effort).

Reply:
Took care of below. Cubeviz now uses the Specviz profile viewer